src/components/Subtitle/SubtitleApp.scss

```
@@ -6,6 +6,9 @@
   width: 100vw;
   border-radius: 8px;
   overflow: hidden;
+  // Defined on this element's inline style by SubtitleApp.tsx (the user's
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)